### ISSUE TITLE
dependency problem on Windows.

### ISSUE THREAD

**Issue by SDX-LV** [jaakkopasanen/Impulcifer#55]
Hello, I am new to this, but by following the exact "Installing" commands on Windows I first get an error from pip. Followed by failed start. See screenshot.
+ Also I wonder if perhaps "requirements.txt" should get updated AutoEq version?

![Impulcifier dependency problem 2021-04-07 213219](https://user-images.githubusercontent.com/17532214/113923833-4ffbac80-97e9-11eb-8181-3dfa4d567b48.png)

Error text:
`Collecting pandas~=0.25.1
  Using cached pandas-0.25.3-cp38-cp38-win32.whl (8.1 MB)
ERROR: Could not find a version that satisfies the requirement tensorflow~=2.0.0 (from autoeq)
ERROR: No matching distribution found for tensorflow~=2.0.0

(venv) C:\Git_Impulcifier\Impulcifer>python impulcifer.py --help
Traceback (most recent call last):
  File "impulcifer.py", line 6, in <module>
    from tabulate import tabulate
ModuleNotFoundError: No module named 'tabulate'

(venv) C:\Git_Impulcifier\Impulcifer>`

Thank you for your Awesome contributions!

**Comment by SDX-LV** [comment]
I actually found the problem on my side.

I had Python 3.8 installed previously, but because the manual says "Python 3.8 doesn't work yet." I also installed Python 3.7.9. But I forgot that I previously installed "virtualenv" using Python 3.8.
So I effectively tried to run virtual environment with Python 3.8 which resulted in the errors as stated above.

Now I can complete all the demo instructions using Python 3.7.9. Still I wonder if the included version of AutoEq is outdated?
